### PR TITLE
[docs] Add docs for the axes properties popover

### PR DESCRIPTION
**Pull request by KaroMourad**
- Added documentation for manual axes range selection
- Removed documentation for steps density selection

### CODE REVIEW

**Comment by KaroMourad** [comment]
Resolves #1918

**Review by roubkar — APPROVED** [review]

File: docs/source/ui/pages/explorers.md [hunk 1]
```diff
@@ -32,11 +32,11 @@ There are two ways you can query metrics and runs
 An overview of what you can do with queried metrics - the modifiers:
 - [Group by tracked parameters](#group-by-any-parameter)
 - [Aggregate grouped metrics](#aggregate-metrics)
-- [Align metrics by time, epoch or custom metric](#x-axis-properties)
+- [Change axes properties (change x-axis alignment, set axes range)](#axes-properties)
```
Comment (gorarakelyan) [inline docs/source/ui/pages/explorers.md:35]: Configure axes properties sounds better to me

File: docs/source/ui/pages/explorers.md [hunk 2]
```diff
@@ -161,30 +144,35 @@ By default, metrics aligned by [Step](#step).
...
-<img src="https://docs-blobs.s3.us-east-2.amazonaws.com/images/ui/pages/metrics-explorer/alignment_absolute.png" style="border-radius: 8px; border: 1px solid #E8F1FC" alt="alignment_absolute" />
+<img src="https://docs-blobs.s3.us-east-2.amazonaws.com/images/ui/pages/metrics-explorer/alignment-absolute.png" style="border-radius: 8px; border: 1px solid #E8F1FC" alt="alignment_absolute" />
 
 ###### Custom Metric
 
 By setting metrics alignment to Custom Metric, x-axis values will represent selected metric values, you can detect correlations between queried metrics and selected metric.
 
+##### Set axes range:
+
+Passing `min` and `max` values for the corresponding axis, will define the axis scale range minimum and maximum bounds (as `[min, max]`) for the chart(s).
```
Comment (gorarakelyan) [inline docs/source/ui/pages/explorers.md:173]: I would go with smth like this `To fix an axis range across all the charts, set the corresponding axis minimum and maximum bounds in the form.`

**Review by gorarakelyan — APPROVED** [review]
lgtm

**Pull request merged**